src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import add from './add';
+
+class HelloWorld extends Component {
```

Comment:
This could also be turned into a functional component, since we are not using state here.